doc/release-notes-30713.md

```diff
@@ -0,0 +1,5 @@
+Updated RPCs
+------------
+
+- The `status` action of the`scanblocks` RPC now returns an additional array `relevant_blocks` containing
+  the matching block hashes found so far during a scan.
```

Comment:
Just a drive-by comment, but there is a typo in the PR title (revelant -> relevant)

Reply:
Thanks, fixed.